View Controllers/FeedViewController.m

```diff
@@ -0,0 +1,132 @@
+//
+//  FeedViewController.m
+//  social-maps
+//
+//  Created by César Francisco Barraza on 7/30/18.
+//  Copyright © 2018 Bevin Benson. All rights reserved.
+//
+
+#import "FeedViewController.h"
+#import "ListAdditionEvent.h"
+#import "FeedEvent.h"
+#import "AdditionFeedCell.h"
+
+@interface FeedViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSArray<FeedEvent *> *events;
+@end
+
+@implementation FeedViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    // set up tableview
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    [self.tableView setRowHeight:64];
+    [self fetchEvents];
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+- (void)fetchEvents {
+    PFQuery *query = [PFQuery queryWithClassName:@"FeedEvent"];
+    PFUser *currentUser = [PFUser currentUser];
+    NSMutableArray<FeedEvent *> *totalEvents = [NSMutableArray new];
+    
+    // get events from current user
+    [query whereKey:@"user" equalTo:currentUser];
+    [query findObjectsInBackgroundWithBlock:^(NSArray * _Nullable objects, NSError * _Nullable error) {
+        if(objects != nil)
+        {
+            for(PFObject *object in objects)
+            {
+                [totalEvents addObject:[[FeedEvent alloc] initWithParseObject:object]];
```

Comment:
hihi! feedback here - when we're initializing the feed events, let's initialize them as the class they actually are (e.g. a ListAdditionEvent) here, rather than in the constructor of FeedEvent